### ISSUE TITLE
Highlight layer on identify initially does not match side menu

### ISSUE THREAD

**Issue by JahidAhmed** [fgpv-vpgf/fgpv-vpgf#3123]
Tested build: http://fgpv.cloudapp.net/demo/develop/prod/samples/index-samples.html?sample=46

Steps to reproduce and example:
- Identify on map
- Side menu shows this:
![image](https://user-images.githubusercontent.com/9637542/47930628-dd14db80-dea2-11e8-8af7-b9532310982f.png)
- Map shows this:
![image](https://user-images.githubusercontent.com/9637542/47930651-e8680700-dea2-11e8-8e80-baa6bdaf12d6.png)
- If we then click on any of the layers in the side menu shown above, it'll correctly display the layer on the map. The problem happens only the first time.
![image](https://user-images.githubusercontent.com/9637542/47930683-059cd580-dea3-11e8-9102-2062b6d7b319.png)

**Comment by AleksueiR** [comment]
I agree - for consistency, only the hits in the initially pre-selected layer should be highlighted on identify. Good catch.
